.pipelines/containerSourceData/memcached/Dockerfile-Memcached

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+ARG BASE_IMAGE
+
+FROM $BASE_IMAGE
+
+@INCLUDE_MAIN_RUN_INSTRUCTION@
+
+USER memcached
```

Comment:
The user should be `memcache` for consistency.

Reply:
Correct. I took your latest changes from the ADO repo and updated this PR.